README.md

```diff
@@ -65,14 +65,17 @@ Install additional build tools:
 
 Install the OAK node from git source:
 
-    
+    git clone git@github.com:OAK-Foundation/OAK-blockchain.git    
 
 Run node on [Dusty Network](https://telemetry.polkadot.io/#list/Dusty):
```

Comment:
Run node on [Dusty Network](https://telemetry.polkadot.io/#list/Dusty):

    oak

What do those lines mean? I think those are typos, or copied from somewhere. Let's remove them.

Reply:
no idea, it's stale...i'll remove